.github/workflows/trivy.yml

```diff
@@ -0,0 +1,23 @@
+name: dsp-appsec-trivy
+on:
+  pull_request:
```

Comment:
Does this mean "yes, do it on pull request" or does an empty section mean skip on PR? (Could use a comment for the uninitiated.)

Reply:
Yep, it means "scan on pull request on any branch". Also it's an OR so either PR or on schedule, it will run both. The last step of uploading results to GH Security Tab only works on schedule, however; results from a PR scan are intended to be checked before merging the PR ;)